lib/shopify_graphql/configuration.rb

```diff
@@ -4,10 +4,12 @@ class Configuration
     attr_accessor :webhook_jobs_namespace
     attr_accessor :webhook_enabled_environments
     attr_accessor :webhooks_manager_queue_name
+    attr_accessor :convert_case
 
     def initialize
       @webhooks_manager_queue_name = Rails.application.config.active_job.queue_name
       @webhook_enabled_environments = ['production']
+      @convert_case = false
```

Comment:
`convert_case` as a boolean isn't super descriptive of what this configuration option is for (converting what to what?). I would suggest either keeping it as a boolean and calling it something like `use_snake_case`, or making it something like `case_style` with a default of `:camel_case` and opt in to `:snake_case`.